bundles/org.openhab.binding.tacmi/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -150,12 +162,20 @@
 		<item-type>Switch</item-type>
 		<label>Switch State (Read-Only)</label>
 		<description>An On/Off state read from C.M.I.</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>Power</tag>
+		</tags>
 		<state readOnly="true"/>
 	</channel-type>
 	<channel-type id="schema-switch-rw">
 		<item-type>Switch</item-type>
 		<label>Switch State (Changeable)</label>
 		<description>A modifiable On/Off state read from C.M.I.</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>Power</tag>
```

Comment:
Just `Control` and `Switch` when allowed

Reply:
I will change to `Switch`